jnosql-mapping/jnosql-mapping-graph/pom.xml

```diff
@@ -31,15 +31,18 @@
         <neo4j.connector.version>0.9-3.4.0</neo4j.connector.version>
     </properties>
     <dependencies>
+        <dependency>
+            <groupId>org.eclipse.jnosql.mapping</groupId>
```

Comment:
@dearrudam, I would remove it.
I would change the necessary for this PR.

Reply:
@otaviojava I figure out that `org.eclipse.jnosql.mapping.test.jupiter` contains classes pretty much similar to provided ones by the `jnosql-junit` module ... could I refactor it to use the `jnosql-junit` module as test dependency?

Reply:
@otaviojava I did it! :smile:

Reply:
Thanks :)